### PR TITLE
https://mosip.atlassian.net/browse/MOSIP-30442  added esignet testcases

### PR DESCRIPTION
**Pull request by anup-nehe**
added testcases for esignet.
downloadAndVerifyVcUsingUinViaEsignet
downloadAndVerifyVcUsingVidViaEsignet
downloadVcUsingUinViaEsignetNoInternateOpenScan
downloadVcViaEsignetAndPinUnpinWithNoNetwork
downloadAndVerifyVcInInvalidEnvForEsignet
downloadAndVerifyVcInNewEnvForEsignet

### CODE REVIEW

File: injitest/src/main/java/inji/pages/CredentialRegistryPage.java [hunk 1]
```diff
@@ -34,6 +40,11 @@ public CredentialRegistryPage setEnterIdTextBox(String env) {
         return this;
     }
     
+    public CredentialRegistryPage setEnterIdToEsignetTextBox(String env) {
```
Comment (kamalsinghthoughtworks) [inline injitest/src/main/java/inji/pages/CredentialRegistryPage.java:43]: Rename method

File: injitest/src/main/java/inji/pages/SetPasscode.java [hunk 2]
```diff
@@ -34,6 +34,19 @@ public ConfirmPasscode enterPasscode(String passcode, Target os) {
         return new ConfirmPasscode(driver);
     }
 
+    public ConfirmPasscode enterPasscodeForEsignet(String passcode, Target os) {
+        char[] arr = passcode.toCharArray();
```
Comment (kamalsinghthoughtworks) [inline injitest/src/main/java/inji/pages/SetPasscode.java:38]: rename arr variable

File: injitest/testng.xml [hunk 3]
```diff
@@ -98,11 +98,15 @@
                     <include name="openCameraOnFlightMode"/>
                     <include name="activateVcWithoutInternet"/>
                     <include name="verifyListOfLanguagesInOfflineMode"/>
-                    <include name="deleteDownloadedVcInOfflineMode"/>
-                    <include name="verifyHelpPageOfflineMode"/>
+		            <include name="deleteDownloadedVcInOfflineMode"/>
```
Comment (kamalsinghthoughtworks) [inline injitest/testng.xml:101]: formatting

**Review by lsivanand — APPROVED** [review]

**Review by vijay151096 — APPROVED** [review]
reviewed by @kamalsinghthoughtworks , approving it.

**Pull request merged**